crates/server/src/server/server_future.rs

```diff
@@ -265,6 +284,7 @@ impl<T: RequestHandler> ServerFuture<T> {
         let ((cert, chain), key) = certificate_and_key;
 
         let handler = self.handler.clone();
+        let acces = self.access.clone();
```

Comment:
looks like a minor type, I bet I ignored my spelling check warnings...

Reply:
oh, it's only on openssl, my guess is that code isn't checked based on the flags. This variable is not being used, there's a warning, but you have to compile with `just dns-over-openssl` otherwise this code is never enabled.